keras_cv/models/__init__.py

```diff
@@ -96,6 +96,12 @@
 from keras_cv.models.mobilenet_v3 import MobileNetV3Small
 from keras_cv.models.object_detection.faster_rcnn.faster_rcnn import FasterRCNN
 from keras_cv.models.object_detection.retina_net.retina_net import RetinaNet
+from keras_cv.models.object_detection.yolo_v8.yolo_v8_backbone import (
+    YOLOV8Backbone,
+)
+from keras_cv.models.object_detection.yolo_v8.yolo_v8_detector import (
+    YOLOV8Detector,
```

Comment:
What do we think about YOLOV8BoxDetector?  Detector is basically just adding noise - is it detecting classes?  segmentation maps?  Boxes?

Know BoxDetector is long but I feel this is very undescriptive currently.